wgpu-core/src/instance.rs

```diff
@@ -843,6 +846,8 @@ pub enum RequestDeviceError {
     Device(#[from] DeviceError),
     #[error(transparent)]
     LimitsExceeded(#[from] FailedLimit),
+    #[error("Failed to initialize Timestamp Normalizer")]
+    TimestampNormalizerInitFailed(#[from] TimestampNormalizerInitError),
```

Comment:
@sagudev I'm getting a little tripped up here - RequestDeviceError implements serialize/deserialize (after https://github.com/gfx-rs/wgpu/pull/5992) but this means I can't include any interesting error types in this error. I'm trying to understand what your exact requirements are and how I might be able to get around this.

Reply:
In servo we send [`RequestDeviceError ` and `RequestAdapterError` to content process via IPC](https://github.com/servo/servo/blob/0051234e2cf4e206256601a7fbd02b38be54d3f1/components/webgpu/ipc_messages/to_dom.rs#L87) (that's why we need serialize/deserialize) and handle error cases there (where we do appropriate promise resolve). But we can always roll our own, actually here we probably should, as we only need 3 cases in content process:
- wrong features
- wrong limits
- other unexpected

So feel free to remove serialize/deserialize from `RequestDeviceError`.

Reply:
Cool, thanks!